drivers/vfio/pci/nvgrace-gpu/main.c

```diff
@@ -1121,6 +1121,7 @@ static void nvgrace_gpu_remove(struct pci_dev *pdev)
 	struct h_node *cur;
 	unsigned long bkt;
 	struct hlist_node *tmp_node;
+
```

Comment:
could you provide the warning of checkpatch? thanks

Reply:
`❮ scripts/checkpatch.pl --no-tree -f drivers/vfio/pci/nvgrace-gpu/*.c
----------------------------------
drivers/vfio/pci/nvgrace-gpu/egm.c
----------------------------------
total: 0 errors, 0 warnings, 497 lines checked

drivers/vfio/pci/nvgrace-gpu/egm.c has no obvious style problems and is ready for submission.
-----------------------------------
drivers/vfio/pci/nvgrace-gpu/main.c
-----------------------------------
WARNING: Missing a blank line after declarations
#1124: FILE: drivers/vfio/pci/nvgrace-gpu/main.c:1124:
+	struct hlist_node *tmp_node;
+	hash_for_each_safe(nvdev->resmem.htbl, bkt, tmp_node, cur, node) {

total: 0 errors, 1 warnings, 1170 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

drivers/vfio/pci/nvgrace-gpu/main.c has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.`

Reply:
thanks